### PR TITLE
upgrade to 0.8-SNAPSHOT

### PR DESCRIPTION
**Pull request by az-qbradley**

### CODE REVIEW

File: src/main/resources/META-INF/squbs-meta.conf [hunk 1]
```diff
@@ -13,3 +13,8 @@ squbs-actors = [
     name = sample
   }
 ]
+squbs-extensions = [
```
Comment (akara) [inline src/main/resources/META-INF/squbs-meta.conf]: This one looks a little odd to you. Why would user code need this?

Comment (az-qbradley) [inline src/main/resources/META-INF/squbs-meta.conf]: they will have a hard shutdown if without

Comment (akara) [inline src/main/resources/META-INF/squbs-meta.conf]: Hmmm, this should not be in user code. We need to take care the shutdown is proper. Is there any use case user code won't want this?

Comment (az-qbradley) [inline src/main/resources/META-INF/squbs-meta.conf]: I am open either way. Unlike rocksqubs, the unicomplex lifecycle events are not observed ... so the kill will terminate app regardless of lifecylce. I can see both points.

Comment (akara) [inline src/main/resources/META-INF/squbs-meta.conf]: Reviewed the shutdown code. You should not need this. It is circular. Please let me know you need a discussion.

Comment (az-qbradley) [inline src/main/resources/META-INF/squbs-meta.conf]: I will remove it, no objection.
Can you clarify on the circular part though?

**Pull request merged**